contracts/Strategy.sol

```diff
@@ -403,6 +365,18 @@ contract Strategy is BaseStrategy {
         }
     }
 
+    event Number(string a, uint256 b);
```

Comment:
remove?

Reply:
Fixed in fixed in bc6343e2f78fe8138a94b7e87c13f455fa2075bf